.env

```diff
@@ -23,6 +23,12 @@ TRUSTED_PROXIES=127.0.0.1,REMOTE_ADDR
 DEFAULT_URI='https://itksites.local.itkdev.dk/'
 ###< symfony/framework-bundle ###
 
+###> symfony/messenger ###
+#MESSENGER_TRANSPORT_DSN='doctrine://default?queue_name=async'
+MESSENGER_TRANSPORT_DSN='amqp://user:password@rabbit:5672/%2f/messages'
+MESSENGER_TRANSPORT_DSN_FAILED='doctrine://default?queue_name=failed'
```

Comment:
Is it deliberately that you store failed messages in the database (and not in RabbitMQ)?

Reply:
yes, we have seen rabbit go down or not respond and thereby having message missed. It's recommend that you use an another storage for failed messages to ensure to be able to replay them.